CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-# Contributing guidelines
+<!-- # Contributing guidelines -->
```

Comment:
Don't edit the contributing guidelines

Reply:
this seems to have happened by mistake, didn't intend to

Reply:
```suggestion
# Contributing guidelines
```
Then please uncomment this line